python/pyspark/ml/classification.py

```diff
@@ -581,8 +602,11 @@ def _create_model(self, java_model):
 
 
 @inherit_doc
-class DecisionTreeClassificationModel(DecisionTreeModel, JavaMLWritable, JavaMLReadable):
+class DecisionTreeClassificationModel(DecisionTreeModel, JavaClassificationModel, JavaMLWritable,
```

Comment:
@holdenk are we not missing out `GBTClassificationModel`, `RandomForestClassificationModel` in classification? I think GBT should just be `JavaPredictionModel`